packages/elements/src/text-field/index.ts

```diff
@@ -232,8 +245,10 @@ export class TextField extends ControlElement {
   private renderIcon (): TemplateResult | null {
     return this.icon ? html`
     <ef-icon
+        role="button"
```

Comment:
Although the icon is not focusable when `iconHasAction=false` and role would not matter but I think it would be semantically better if we put a condition here as well just like `aria-label`.